src/DISPUTE_MANAGEMENT.md

```diff
@@ -0,0 +1,19 @@
+# Dispute Management
+
+The purpose of this document is to provide guidelines to Product Teams in regards to how to settle disputes about situations that may arise when working with the Auro Design System.
+
+### The team's Product Designer submitted comps that contradict the specification of the Auro Design System.
+
+It is recommended that designers leverage Auro resources as much as possible. But there will be times when the design asset provided does not provide the assistance the designer needs to meed the goal of the new product work. 
+
+It is recommended that the discovered inconsistency is reviewed by the members of the Product Team to understand the reason for the inconsistency achieve a resolution. 
+
+If a resolution cannot be found within the current constraints of the design or engineering assets made available from the Auro Team, it is suggested that the Product Team consult with the Auro team for assistance. 
+
+Resolution may come in the form of augmenting the support of an existing asset, the creation of a new asset or the determination that the inconsistency is to be resolved in another way. 
+
+### A Team's engineer discovers a bug with an existing asset.
+
+In the rare case that the delivered Auro asset is not living up to its specification, then it is recommended that the team submit an issue with the specified repository. 
```

Comment:
**Unnecessary word**: "then" is not necessary for: "it is recommended that the team submit an issue with the specified repository"